### ISSUE TITLE
~ and $HOME fail for key path

### ISSUE THREAD

**Issue by brandondrew** [pearkes/tugboat#130]
I configured `~/.tugboat` using `$HOME` in the path to my private SSH key, but it didn't work:
~~~
$ tugboat ssh rethink
Droplet fuzzy name provided. Finding droplet ID...done, 3346933 (rethink)
Executing SSH (rethink)...
Warning: Identity file $HOME/.ssh/id_rsa_digitalocean not accessible: No such file or directory.
root@104.236.8.39's password:
~~~
Then I edited `~/.tugboat` to use `~` in the path, and it still didn't work
~~~
$tugboat ssh rethink
Droplet fuzzy name provided. Finding droplet ID...done, 3346933 (rethink)
Executing SSH (rethink)...
Warning: Identity file ~/.ssh/id_rsa_digitalocean not accessible: No such file or directory.
root@104.236.8.39's password:
~~~
Then I used an absolute path, and it worked:
~~~
$ tugboat ssh rethink                                        15544  12-01 22:14 EST  ~  tintin
Droplet fuzzy name provided. Finding droplet ID...done, 3346933 (rethink)
Executing SSH (rethink)...
Welcome to Ubuntu 14.04.1 LTS (GNU/Linux 3.13.0-40-generic x86_64)

 * Documentation:  https://help.ubuntu.com/

  System information as of Mon Dec  1 22:06:56 EST 2014

  System load: 0.0                Memory usage: 9%   Processes:       52
  Usage of /:  14.9% of 19.56GB   Swap usage:   0%   Users logged in: 0

  Graph this data and manage this system at:
    https://landscape.canonical.com/

Last login: Mon Dec  1 21:22:21 2014 from 108-243-16-73.lightspeed.gdrpmi.sbcglobal.net
root@rethink:~#
~~~

Since the authorization setup mentions its default path with a `~`, this is surprising and unintuitive.  It would be best to allow the use of `$HOME` and `~`, but if there's some major obstacle to doing that, then it would be better if the default would be mentioned with an absolute path, and perhaps the requirement for an absolute path should be mentioned.

**Comment by brandondrew** [comment]
As mentioned in #146 it should be sufficient to allow `~` instead of all environment variables.

**Comment by petems** [comment]
Fixed in #146
